charts/drupal/values.yaml

```diff
@@ -575,6 +575,22 @@ memcached:
     # MaxItemSize
     - -I 4M
 
+# https://github.com/bitnami/charts/blob/master/bitnami/redis/values.yaml
+redis:
+  architecture: standalone
```

Comment:
Should we add the `enabled: false` line here?

Reply:
Good idea, fixed!

Reply:
and then i set it to `enabled: true` :facepalm:

Reply:
false alarm, it was true only in chart tests